Sources/Lotsawa/UniqueCounter.swift

```diff
@@ -1,6 +1,10 @@
+/// A function object; returns the number of unique `T`'s in a given Sequence.
+///
+/// Implemented as a function object to optimize repeated invocations by reusing a Set<T>.
```

Comment:
That's not API documentation so not a doc comment I think; frankly I don't know ATM why I didn't use a closure with a captured `Set`.  The above doesn't quite explain it by itself.